mlflow/server/js/src/common/utils/FeatureUtils.ts

```diff
@@ -39,18 +38,43 @@ export const shouldEnableImageGridCharts = () => false;
  * Log table images feature
  */
 export const shouldEnableLogTableImages = () => false;
-
+/**
+ * Manual range controls feature
+ */
+export const shouldEnableManualRangeControls = () => false;
+/**
+ * Keeping original lines when smoothing.
+ */
+export const shouldEnableChartsOriginalLinesWhenSmoothing = () => false;
+/**
+ * Enable to use larger color selection (palette of 400 colors) and murmur hashing of
+ * run UUIDs for color assignment
+ */
+export const shouldEnableLargerColorSelection = () => false;
+/**
+ * Enable chart expressions feature
+ */
+export const shouldEnableChartExpressions = () => false;
 /**
  * Should enable toggling aggregation of individual runs in run groups
  */
 export const shouldEnableToggleIndividualRunsInGroups = () => false;
 
+/**
+ * Enables toggle controls for hiding charts with no data
+ */
+export const shouldEnableHidingChartsWithNoData = () => false;
+
 /**
  * Update relative time axis to use date
  */
 export const shouldEnableRelativeTimeDateAxis = () => false;
 
 export const shouldEnableTracingUI = () => true;
+export const shouldEnableRunDetailsPageTracesTab = () => false;
+export const shouldUseCompressedExperimentViewSharedState = () => true;
+export const shouldEnableUnifiedChartDataTraceHighlight = () => false;
+export const shouldDeferLineChartRendering = () => false;
```

Comment:
I think those are fine to be enabled. Just tested it and looks ok.
```suggestion
export const shouldEnableUnifiedChartDataTraceHighlight = () => true;
export const shouldDeferLineChartRendering = () => true;
```